### PR TITLE
chore(docs): cancellations

### PR DESCRIPTION
**Pull request by deebloo**
addresses cancellation question in #143

### CODE REVIEW

File: docs/advanced/cancelation.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export class ZooState {
+  constructor(private animalService: AnimalService) {}
+
+  @Action(FeedAnimals)
+  get({ getState, setState, dispatch }, { payload }) {
+    return this.animalService.get(payload).pipe(
+      tap((res) => {
+        setState(res)
+      }),
+      takeUntil(this.actions.pipe(ofAction(RemoveTodo)))
```
Comment (yarrgh) [inline docs/advanced/cancelation.md]: Where does `this.actions` come from? Typescript complains:
> Property 'actions' does not exist on type 'ZooState'

Comment (deebloo) [reply docs/advanced/cancelation.md]: import it from @ngxs/store

Comment (deebloo) [reply docs/advanced/cancelation.md]: ah I didn't inject it in the example. will do

**Comment by amcdnl** [comment]
This just shows how powerful of a platform we've got now!

**Pull request merged**